DEVELOPMENT.md

```diff
@@ -170,7 +170,7 @@ Next, run:
 
 ```shell
 ko apply -f config/
-./hack/dev-patch-config-gke.sh my-k8s-cluster-name  # optional
+PROJECT_ID="my-gcp-project-id" ./hack/dev-patch-config-gke.sh my-k8s-cluster-name  # optional
```

Comment:
Perhaps: `PROJECT_ID=$(gcloud config get-value project)`?

Reply:
@vagababov I think this is better. I just used it and it wasn't a project in the same cluster. I think people can spell out their project-id just fine. :)